src/Altinn.Platform/Altinn.Platform.Storage/Storage/Program.cs

```diff
@@ -63,13 +65,6 @@ public static void Main(string[] args)
                 logging.AddProvider(new SerilogLoggerProvider(logger));
             });
 
-        // Parameters required for integration with SBL in local development             
```

Comment:
These lines cannot be deleted.. We require them for the integration with SBL-images when developing and testing image to image.

Reply:
Added again. I didn't find the best place for them so I left them where they were.